app/components/input-field/press-release-publication-channels.js

```diff
@@ -36,6 +47,10 @@ export default class InputFieldPressReleasePublicationChannelsComponent extends
     } else {
       selectedPublicationChannels.addObject(publicationChannel);
     }
+    if (selectedPublicationChannels.length && !selectedPublicationChannels.includes(this.subscribersFlandersBe)) {
```

Comment:
Net voor deze `if` misschien een comment toevoegen zoals
```
// if any publication-channel is selected, website Flanders publication-channel must be automatically added
```
zodat we in de toekomst nog weten waarom we dit doen